spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,139 @@
+          :email=>"NEVADAAGENTOFFICE@DOR.MO.GOV",
+          :agent=>"THE IRON VAULT, LLC",
+          :officemanager=>"ELIZABETH CASSIDY",
+          :daysopen=>
+           "Monday - Friday - 8:00 to 5:00, Last Saturday of the Month  - 8:00 to 12:00",
+          :daysclosed=>"Monday - Friday from 1:00 to 1:30",
+          :holidaysclosed=>"Christmas Day (12/25/23), New Year's Day (1/1/24), Martin Luther King Jr. Day (1/15/24), Lincoln's Birthday (2/12/24), President's Day (2/19/24), Truman's Birthday (5/8/24), Memorial Day (5/27/24), Juneteenth (6/19/24), Independence Day (7/4/24), Labor Day (9/2/24), Columbus Day (10/14/24), Veterans Day (11/11/24),
+            Thanksgiving Day (11/28/24), Christmas Day (12/25/24)",
+          :additionaldaysclosed=>
+           "3/6/2023 (at 3:45 PM ),    3/6/2023 (at 9:30 AM until 12:00 PM ),    3/6/202
+          3 (open at 12:00 PM ),  7/1/2023,   7/3/2023,     11/24/23,        11/25/23, 11/
+          29/2024, 12/24/2024, ",
+          :latlng=>{:latitude=>"37.837063", :longitude=>"-94.347818"},
+          :facebook_url=>"https://m.facebook.com/nevadalicenseoffice",
+          :additional_license_office_info=>
+           "MOTOR VEHICLE PHONE IN RENEWAL - (417) 448-5705",
+          :":@computed_region_ny2h_ckbz"=>"313",
+          :":@computed_region_c8ar_jsdj"=>"73",
+          :":@computed_region_ikxf_gfzr"=>"2218"}]
+    end
+
+    it 'is an instance of FacilityFactory' do
+       expect(@facility_factory).to be_a(FacilityFactory)
+    end
+
+    it 'creates CO facility objects from api data' do
+       co_facility_locations = DmvDataService.new.co_dmv_office_locations
+       co_facilities = @facility_factory.create_facilities(co_facility_locations)
+       co_facility_1 = co_facilities.first
+
+       expect(co_facilities).to be_an(Array)
+       expect(co_facilities).to all be_a(Facility)
+       expect(co_facility_1.name).to be_a(String)
+       expect(co_facility_1.phone).to be_a(String)
+       expect(co_facility_1.address).to be_a(String)
+    end
+
+    it 'formats CO data correctly' do
+       formatted_facility = @facility_factory.create_facilities(@co_sample)
+
+       expect(formatted_facility.first.name).to eq("DMV Tremont Branch")
+       expect(formatted_facility.first.address).to eq("2855 Tremont Place Suite 118 Denver CO 80205")
+       expect(formatted_facility.first.phone).to eq("(720) 865-4600")
+    end
+
+    it 'creates NY facility objects from api data' do
+        new_york_facilities = DmvDataService.new.ny_dmv_office_locations
+        ny_facilities = @facility_factory.create_facilities(new_york_facilities)
+        ny_facility_1 = ny_facilities.first
+
+        expect(ny_facilities).to be_an(Array)
+        expect(ny_facilities).to all be_a(Facility)
+        expect(ny_facility_1.name).to be_a(String)
+        expect(ny_facility_1.phone).to be_a(String)
+        expect(ny_facility_1.address).to be_a(String)
+    end
+
+    it "formats NY data correctly" do
```

Comment:
This is super robust testing! Awesome! In my opinion, it's also okay to just do one of these -- test for the exact values (since you have them) OR if you were just calling the API and weren't sure of the exact values, you could test for the correct data types in each of these fields. You don't necessarily need to do both.